test/net/sourceforge/kolmafia/textui/command/FaxbotCommandTest.java

```diff
@@ -31,10 +33,32 @@ public static void beforeAll() {
     // Set to true so we make http requests
     ChatManager.setChatLiteracy(true);
 
+    // This needs to happen before database configuration.
+    // ignoring cleanup
+
+    var ignoredCleanup =
```

Comment:
if you store the cleanup in the class (statically) you can call `close` on it in `afterAll`.

Reply:
Done.  Thanks for the suggestion.